.github/actions/run-e2e-test/action.yml

```diff
@@ -26,6 +26,15 @@ inputs:
   follow-up-finalization-check:
     description: 'Whether to run a follow-up finalization check.'
     required: false
+  upgrade_version:
+    description: 'Version which will be used during the VersionUpgrade test.'
+    required: false
+  upgrade_session:
+    description: 'Session for which an upgrade should be scheduled.'
+    required: false
+  upgrade_finalization_wait_sessions:
+    description: 'Number of sessions we should wait after performing an upgrade.'
+    required: false
```

Comment:
I have a feeling that this is not the best way of introducing parametrization... if I'm not mistaken, these parameters will be used in only just a few tests that focus solely on upgrading. Attaching them to the common parameters set seems questionable.

I'm afraid that it the only alternative is to pass such values as env variables directly in the job specification. Neither pretty nor elegant, but still, that would be my suggestion.